primitives/linux-port/column.cpp

```diff
@@ -1240,6 +1240,14 @@ void filterColumnData(
     T Min = datatypes::numeric_limits<T>::max();
     T Max = (KIND == KIND_UNSIGNED) ? 0 : datatypes::numeric_limits<T>::min();
 
+   if (KIND == KIND_TEXT)
```

Comment:
It would be better to have a template function that assigns/returns Min/Max having <T, KIND> as its params and write couple SFINAE versions of the function.

Reply:
After some consideration I arrived to the conclusion I can fix this myself.